sdk/ml/azure-ai-ml/azure/ai/ml/operations/_component_operations.py

```diff
@@ -217,6 +220,43 @@ def get(self, name: str, version: Optional[str] = None, label: Optional[str] = N
         :return: The specified component object.
         :rtype: ~azure.ai.ml.entities.Component
         """
+        return self._get(name=name, version=version, label=label)
+
+    @experimental
+    @monitor_with_telemetry_mixin(logger, "Component.Get", ActivityType.PUBLICAPI)
+    def download(
+        self,
+        name: str,
+        output_dir: Union[str, Path, PathLike],
```

Comment:
Let's follow existing API design, like model download

Reply:
move `version` behind `download_path` as download component without `version` provided is possible.